Framework/Kernel/CMakeLists.txt

```diff
@@ -594,7 +594,7 @@ set(
   ${ExternalData_BINARY_ROOT}/Testing/Data/UnitTest;${ExternalData_BINARY_ROOT}/Testing/Data/DocTest;${MANTID_ROOT}/instrument
   )
 set(COLORMAPS_FOLDER ${MANTID_ROOT}/installers/colormaps/)
-set(MANTIDPUBLISHER "https://upload.mantidproject.org/scriptrepository?debug=1")
+set(MANTIDPUBLISHER "https://upload.mantidproject.org/scriptrepository")
```

Comment:
Can we leave this as `?debug=1` for developer builds. It prevents accidentally modifying the real repo and pushes the updates to https://github.com/mantidproject/sandbox.

Reply:
Oh that makes sense, I hadn't realised that, it just looked like it wasn't uploading. I'll change it back

Reply:
For developer builds you can add a line to the `Mantid.user.properties` file to change it temporarily for production if required.